ops/terraform/modules/resources/iam/main.tf

```diff
@@ -56,3 +66,44 @@ resource "aws_iam_role_policy_attachment" "cloudwatch_agent_policy_attachment" {
   role       = aws_iam_role.instance.id
   policy_arn = data.aws_iam_policy.cloudwatch_agent_policy.arn
 }
+
+
+resource "aws_iam_policy" "ssm" {
+  name        = "bfd-${var.env_config.env}-${local.service}-ssm-parameters"
+  description = "Permissions to /bfd/${var.env_config.env}/common/nonsensitive, /bfd/${var.env_config.env}/${local.service} SSM hierarchies"
+  policy      = <<-EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ssm:GetParametersByPath",
+        "ssm:GetParameters",
+        "ssm:GetParameter"
+      ],
+      "Resource": [
+
+        "arn:aws:ssm:us-east-1:${data.aws_caller_identity.current.account_id}:parameter/bfd/${var.env_config.env}/common/nonsensitive/*",
+        "arn:aws:ssm:us-east-1:${data.aws_caller_identity.current.account_id}:parameter/bfd/${var.env_config.env}/${local.service}/*"
```

Comment:
Just to be clear, this would allow anyone with this policy access to both sensitive and non-sensitive data. This may be by design, but thought I would call it out. 
> If a user has access to a path, then the user can access all levels of that path. For example, if a user has permission to access path /a, then the user can also access /a/b. Even if a user has explicitly been denied access in IAM for parameter /a/b, they can still call the GetParametersByPath API operation recursively for /a and view /a/b.

Reply:
Thanks for the feedback!

> [...] this would allow anyone with this policy access to both sensitive and non-sensitive data.

Yes, exactly! The instances running the services will have access to common, nonsensitive values as well as service-specific nonsensitive **and** sensitive values. We can chat offline about further security implications of this if you'd like. Thanks again!